CMakeLists.txt

```diff
@@ -507,6 +507,14 @@ else ()
   set (tgt_file_ext "-shared")
 endif ()
 
+#-----------------------------------------------------------------------------
+# perl is used in some optional src and tests, check availability
+find_package (Perl)
+if (PERL_FOUND)
+  set (H5_PERL_FOUND YES)
+endif ()
+#-----------------------------------------------------------------------------
+
```

Comment:
Why can't we just use PERL_FOUND directly?

Reply:
@byrnHDF note this change is also in the PRs #523, 531, and 548 to merge long double changes to 1.12, 1.10, and 1.8, so I'm waiting for this answer to merge those, too.

Reply:
> Why can't we just use PERL_FOUND directly?

There seemed to be a variable visibility issue, which these changes solved. I did not do any further investigating.